types/evernote/index.d.ts

```diff
@@ -8,6 +8,11 @@
 
 import { oauth1tokenCallback, OAuth } from 'oauth';
 
+export {};
+type Dictionary<TKey extends number | string, TValue> = {
+    [key in TKey]: TValue
+};
+
```

Comment:
```suggestion
export {};
type Dictionary<TKey extends number | string, TValue> = {
    [key in TKey]: TValue
};

```
if you follow my suggestions you will not need this (btw: the type could be just autoexported from within namespace here, no need for `export {}`)